### DISCUSSION TITLE
How do Noop organizations work?

### DISCUSSION THREAD

**Discussion by nprimmer** [noop-inc/documentation#31] [Questions] [answered]
- Are Noop organizations a 1:1 relationship with GitHub organizations?
- How does one onboard a GitHub organization to Noop?
- Can a Noop organization be scoped to an individual GitHub user?

**Comment by milin82** [comment, marked as answer]
- No, Noop organizations are not a 1:1 relationship with Github organizations. **Noop Organization** is a top-level account construct where users can collaborate on running a shared set of Applications. Organizations currently allow user & team management, permissions and billing. 
- **Noop Cloud** is currently in Preview. In the preview phase, new Organization creation and user onboarding is handled manually by the Noop Engagement Team. If you are already a Noop User, you can request creation of a new Organization or adding a new User to an existing organization by using the **Engagement Center**. For first time users, we will soon have a form to request access to Cloud Preview.
- Yes, an Organization can just be one user
